CMakeLists.txt

```diff
@@ -29,8 +35,14 @@ set(RADIX_LIBRARIES ${OPENGL_gl_LIBRARY})
 
 find_package(PkgConfig REQUIRED)
 pkg_search_module(EPOXY REQUIRED epoxy)
```

Comment:
Here is Epoxy required but you want to have it optional when you want to use glad